Openshift4/helm/openshift-artifactory-ha/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # JFrog  Openshift Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [107.27.9] -  Dec15, 2021
```

Comment:
Is the date here related to an older release? Shouldn't this be today or in near future?

Reply:
This is the date operator image is published to redhat registry.Actual release to redhat operatorhub catalog which happens through bundle image is from different repo https://github.com/redhat-openshift-ecosystem/certified-operators